lib/app_config.dart

```diff
@@ -29,8 +30,19 @@ class AppConfig extends StatelessWidget {
   /// AppConfig constructor
   const AppConfig({Key? key}) : super(key: key);
 
+  void _setSystemNavigationBarColor(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(
+      SystemUiOverlayStyle(
+        systemNavigationBarColor:
```

Comment:
```suggestion
        systemNavigationBarIconBrightness: Brightness.dark,
        systemNavigationBarColor:
```

Made the system buttons more visible.

Reply:
"Resolved in [db5ab9b](https://github.com/healthcare-it-solutions/picos/pull/133/commits/db5ab9b1e94088438b03a1bd060ea2a6bd25640b)"